src/pages/student/courses/[courseId]/index.tsx

```diff
@@ -1,29 +1,138 @@
+import MarkdownRender from 'components/markdownRender'
 import { useRouter } from 'next/router'
 import { trpc } from 'utils/trpc'
-import CourseVisualizer from 'components/student/courses/[courseId]/courseVisualizer';
-import IUserAnswerStatisticRepository from 'server/repositories/iUserAnswerStatisticRepository';
-import Module from 'server/entities/module';
-import UserAnswerStatistic from 'server/entities/userAnswerStatistic';
-import User from 'server/entities/user';
+import User from 'server/entities/user'
+import UserAnswerStatistic from 'server/entities/userAnswerStatistic'
 
-
-function Course() {
+function Course(user?: User, userAnswerStatistic?: UserAnswerStatistic[]) {
```

Comment:
```suggestion
type CourseProps = {
  user?: User;
  userAnswerStatistic?: UserAnswerStatistic[]
}
function Course({user, userAnswerStatistic }: CourseProps ) {
```
Componente react só consegue receber um parametro, que é apelidado de "props", temos que pegar os dados de dentro dele fazendo desestruturação